service/src/main/java/com/ge/predix/acs/privilege/management/dao/GraphGenericRepository.java

```diff
@@ -69,10 +69,10 @@ public void deleteInBatch(final Iterable<E> entities) {
     @Override
     public List<E> findAll() {
         try {
-            return this.graph.traversal().V().has(ZONE_ID_KEY).has(getEntityIdKey()).toList().stream()
+            return this.graphTraversal.V().has(ZONE_ID_KEY).has(getEntityIdKey()).toList().stream()
                     .map(this::vertexToEntity).collect(Collectors.toList());
         } finally {
-            this.graph.tx().commit();
+            this.graphTraversal.tx().commit();
```

Comment:
@anurag-gujral: @sanjeevchopra mentioned that there might be nuances to the calls in the `try` block that warrant committing the transaction. In a future story, we can investigate removing the `finally` blocks everywhere this pattern occurs and ensure concurrency-specific tests pass.